cfgov/hmda/templates/orange-explorer.html

```diff
@@ -0,0 +1,2287 @@
+<!DOCTYPE html>
```

Comment:
This HTML file was created by scraping https://www.consumerfinance.gov/data-research/hmda/explore and inlining all the CSS and JS to make it a single standalone file with no external dependencies.